waspc/src/Wasp/Util/IO.hs

```diff
@@ -95,6 +97,9 @@ doesFileExist = SD.doesFileExist . SP.fromAbsFile
 readFile :: Path' Abs (File f) -> IO String
 readFile = P.readFile . SP.fromAbsFile
 
+readFileBytes :: Path' Abs (File f) -> IO B.ByteString
+readFileBytes = B.readFile . SP.fromAbsFile
```

Comment:
Hm, I am not sure. It is not really under the hood, you are returning a type that is Lazy ByteString, that type is not the same as Strict ByteString, so there won't be confusion later. But in general, lazy IO is source of errors often as it is a bit weird to grasp. So I would potentially go with strict as default. At the end, depends where it is used. I wonder now about that readFile above we have -> that is also probably lazy? Ha I am not sure. Actually yeah it likely is, because below we have readFileStrict.
Ok, actually I think what you did is fine. Usually IO is assumed to be lazy unless you add Strict or `'` to it, so this sounds good. As I said, type system should catch this I think, to some degree, I am not sure to which degree but I know haskell complains when I try to mix lazy and strict ByteString (or did I remember that wrong? I don't think so).